eventstore/storethread.go

```diff
@@ -0,0 +1,145 @@
+package eventstore
```

Comment:
Refer to `README.md` to understand this component responsibility (in short: glue between Store and Threads)